4_other_sources/out/norwest_raw_files.yml

```diff
@@ -0,0 +1,3 @@
+4_other_sources/out/norwest_raw_temp_data.feather: 148b6ac52e4a6f9a84eb2c4e666fba17
+4_other_sources/out/norwest_raw_site_data.rds: dc8983d1164d5d0b8f3578877c172d04
```

Comment:
You mentioned debating between storing the hashes of the data files or the hashes of the indicator files. Given that you're converting these file names to indicator files later, it might make sense to hash the indicator files here to avoid that ad hoc string manipulation.

Reply:
Is this file obsolete now that you've changed the output of `do_data_file_tasks()` to an object?